From 95e8c6c597da41edfdda1190d2c0e0cdeedf84bd Mon Sep 17 00:00:00 2001
From: Michael Dickens <mlk@alum.mit.edu>
Date: Mon, 24 Mar 2014 18:03:11 -0400
Subject: [PATCH 08/13] Make compilers happy by adding in a proper return
 instruction to the pycallback_object fall-through
 converter, even though it will never actually be
 issued.

---
 gnuradio-runtime/include/gnuradio/pycallback_object.h |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/gnuradio-runtime/include/gnuradio/pycallback_object.h b/gnuradio-runtime/include/gnuradio/pycallback_object.h
index 756067d..709ca0e 100644
--- a/gnuradio-runtime/include/gnuradio/pycallback_object.h
+++ b/gnuradio-runtime/include/gnuradio/pycallback_object.h
@@ -152,6 +152,9 @@ private:
   myType pyCast(PyObject* obj) {
     printf("TYPE NOT IMPLEMENTED!\n");
     assert(0);
+    // the following is to make compilers happy only.
+    myType dummy;
+    return(dummy);
   };
 
   std::vector<boost::any> d_rpc_vars; // container for all RPC variables
-- 
1.7.10.4

